src/nanoemoji/config.py

```diff
@@ -103,6 +103,14 @@
     None,
     "A program that takes a list of filenames and outputs a file csv whose rows contain filename, codepoint(s), glyph name.",
 )
+flags.DEFINE_integer(
+    "bitmap_resolution", None, "Resolution of bitmap in pixels. Always square for now."
+)
+flags.DEFINE_integer(
+    "bitmap_advance",
+    None,
+    "Advance for bitmap, in pixels. Typically >= bitmap_resolution.",
```

Comment:
we don't need to add this option. We can derive the bitmap advance width from the hmtx width (which can be configured already). 

(ascender - descender) : width == bitmap_resolution : bitmap_width

bitmap_width = bitmap_resolution * width / (ascender - descender)

For our current defaults, this means:

bitmap_width = 128 * 1275 / 1200 == 136

Reply:
Oh nice, good point. Fixed!

Reply:
Done